Vagrantfile

```diff
@@ -77,11 +78,12 @@ end
 save_config
 
 Vagrant.configure(2) do |config|
-  config.vm.box = 'ubuntu-22.04-amd64'
+#  config.vm.box = 'boxen/ubuntu-22.04'
```

Comment:
the goal of this repo is to use my own base images (I also have a debian-12 base image), so please drop these two changes.